sender.py

```diff
@@ -0,0 +1,15 @@
+from configuration import *

```

Comment:
method name is not in sync with content within, the file contains target details and not source...